examples/server/main.go

```diff
@@ -43,7 +45,11 @@ func main() {
 	if err != nil {
 		logger.Fatalf("failed to listen: %v", err)
 	}
-	grpcServer := grpc.NewServer()
+
+	// attempt to fix ENHANCE_YOUR_CALM error when using linkerd
+	grpcServer := grpc.NewServer(grpc.KeepaliveEnforcementPolicy(
+		keepalive.EnforcementPolicy{MinTime: time.Millisecond, PermitWithoutStream: true},
```

Comment:
Is this setting the keep alive to 1ms?

Reply:
It sets minimum allowed time between client's pings to 1ms

Reply:
Do we know why it needs to be set so low? Is it linkerd muxing lots of clients onto a single connection, making their pings all register under the same counter or something?